docs/pages/management/guides.mdx

```diff
@@ -1,10 +1,8 @@
 ---
-title: Setup Guides
-description: Teleport Installation and Configuration Guides.
+title: Integrations
+description: Miscellaneous guides for integrating Teleport with third-party tools.
```

Comment:
I suggest adding a more verbose version of this description to the top of the page body.